.github/workflows/CI.yml

```diff
@@ -0,0 +1,31 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        node-version: [14.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ matrix.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ matrix.os }}-yarn-
+      - run: yarn --frozen-lockfile
+      - run: yarn build
+      - run: yarn lint
+      - run: yarn coverage
```

Comment:
Works fine as a first CI automation step.
I think in the long run we should probably split lint, test and coverage or at least lint and coverage to multiple jobs and probably include workflow badges on the repo readme. 
But thats optional.

Reply:
I fully intend on having a coverage badge when we get closer to 💯 at the very least.